WordPress/src/main/res/layout/publicize_list_fragment.xml

```diff
@@ -113,6 +113,28 @@
 
             </RelativeLayout>
 
+            <com.google.android.material.button.MaterialButton
```

Comment:
You could include the badge layout here [like here](https://github.com/wordpress-mobile/WordPress-Android/blob/eecafadb34d971a67cb5a00511bec6554e7673c9/WordPress/src/main/res/layout/me_fragment.xml#L305)

Reply:
Thanks for the suggestion Ravi. Originally, I had tried using the existing footer as an include, but I ran into a weird issue in dark mode where the button was not working (unless I tapped the surrounding area instead). I did not get to the root of that issue, but using the material button directly seems to resolve the issue. 🤷‍♂️ 

Fwiw, I tried two variations of include before using the button directly: https://github.com/wordpress-mobile/WordPress-Android/pull/17035/commits/02d8c869fca12330b7d8069414641e55a7bff0b1 and https://github.com/wordpress-mobile/WordPress-Android/pull/17035/commits/337af9e05d8b28aa24fc60e25e4c9d3121b9e0f5